docs/docs/advanced/deploy-eliza-to-tee-with-oasis-rofl.md

```diff
@@ -0,0 +1,257 @@
+---
+sidebar_position: 18
+---
+
+# 🛡️ Deploying Eliza to TEE with Oasis ROFL
+
+![](/img/eliza_in_tee.jpg)
+
+## Overview
+
+Eliza can be deployed to a Trusted Execution Environment (TEE) using the [Oasis SDK](https://github.com/oasisprotocol/oasis-sdk). This guide will walk you through the process of setting up and running Eliza in a TEE environment using the Oasis ROFL tool.
+
+### Background
+
+The Oasis ROFL tool is built on top of the [Oasis SDK](https://github.com/oasisprotocol/oasis-sdk), which is designed to simplify the deployment of applications to TEE environments. The main features include:
+
+- Secure deployment of applications to TEE environments
+- Remote attestation and verification
+- Support for both development and production environments
+- Integration with the Oasis network
+
+---
+
+## Core Components
+
+Eliza's TEE implementation consists of several key components that work together to provide a secure execution environment:
+
+1. ROFL Application Configuration
+2. Secret Management
+3. Build and Deployment System
+
+These components ensure:
+- Secure execution of the Eliza agent
+- Protected handling of sensitive data
+- Verifiable proof of TEE execution
+- Support for both development and production environments
+
+---
+
+### ROFL Application Configuration
+
+The ROFL application configuration handles the setup and management of the TEE environment. It supports:
+
+- Multiple deployment modes:
+    - `testnet`: For development and testing
+    - `mainnet`: For production deployment
+
+Key features:
+- Secure application deployment
+- Configuration management
```

Comment:
Configuration management is probably deployment mode above?
Also I suggest you mention Docker compose below to ring a bell for those who know it.

```suggestion
- Secure application deployment
- Raw or Docker compose-based container management
```